.github/workflows/developer_onboarding_notification.yml

```diff
@@ -70,4 +70,4 @@ jobs:
                 issue_number: prNumber,
                 body: message
               });
-            }
+            }
```

Comment:
A new line is required at the end of the file, which is why there is this red symbol here.